src/filter.rs

```diff
@@ -1 +1,71 @@
+/// An abbreviation for the filter checks.
```

Comment:
You have a typo in your commit message. Please fix.

Reply:
As soon as this Pull Request is accepted, it will become invisible in your repository automatically when using squashing as merge strategy.